src/cache_client.rs

```diff
@@ -161,31 +165,182 @@ impl CacheClient {
         request.send(self).await
     }
 
+    /// Sets an item in a Momento Cache
+    ///
+    /// # Arguments
+    ///
+    /// * `cache_name` - name of cache
+    /// * `key` - key of the item whose value we are setting
+    /// * `value` - data to stored in the cache item
+    /// * `ttl` - The TTL to use for the
```

Comment:
Should this be in an optional arguments section?

Reply:
oh good catch, will fix, thanks

Reply:
fixed in https://github.com/momentohq/client-sdk-rust/pull/185/commits/a82609ceedbd3336bc109c3f2b3454a1e1d9dcad , ty